tests/integration/connect-test.js

```diff
@@ -71,6 +73,29 @@ test('stateToComputed can be used with component level CP if notifyPropertyChang
   assert.equal(this.$('.dyno').text(), 'name: Tom', 'should render new value when local component CP changed and notifyPropertyChange invoked');
 });
 
+test('stateToComputed is not invoked extraneously', function(assert) {
+  let callCount = 0;
+  const stateToComputed = () => {
+    callCount++;
+    return { callCount };
+  }
+  this.register('component:test-component', connect(stateToComputed)(Component.extend({
+    layout: hbs`{{callCount}}`
+  })));
+
+  this.render(hbs`{{test-component attr=attr}}`);
+  assert.equal(this.$().text(), '1');
+  assert.equal(callCount, 1);
```

Comment:
@brettburley love to see you adding this test back to the mix - I feel it's a good one to have regardless :)